internal/request/graphql/schema/generate.go

```diff
@@ -1394,6 +1369,33 @@ func isNumericArray(list *gql.List) bool {
 		list.OfType == gql.Float
 }
 
+func genFilterOperatorName(fieldType gql.Type) string {
+	list, isList := fieldType.(*gql.List)
+	if isList {
+		fieldType = list.OfType
+	}
+	if !gql.IsLeafType(fieldType) {
+		return genTypeName(fieldType, filterInputNameSuffix)
+	}
+	notNull, isNotNull := fieldType.(*gql.NonNull)
+	if isNotNull {
+		fieldType = notNull.OfType
+	}
+	switch {
+	case fieldType.Name() == "JSON":
+		return fieldType.Name()
+
+	case isList && isNotNull:
+		return "NotNull" + fieldType.Name() + "ListOperatorBlock"
```

Comment:
thought: It looks like we have a name clash here - I've created a ticket - would you mind adding a todo comment linking to the ticket, and then amend the issue-description to note taht there is a code-todo? 

https://github.com/sourcenetwork/defradb/issues/3123

Reply:
done

Reply:
thanks :)